docs/resources/server/recipes/ingredients.md

```diff
@@ -30,7 +30,7 @@ Though they are functionally identical, compound ingredients replaces the way on
 
 #### DataComponentIngredient
 
-`DataComponentIngredient`s compare the item, damagage, and data components on an `ItemStack`. When `strict` is `true`, the `ItemStack` is checked for an exact match. Otherwise, if only the components specified match, the ingredient will pass. This can be used by specifying the `type` as `neoforge:components`.
+`DataComponentIngredient`s compare the item, damagage, and [data components][datacomponents] on an `ItemStack`. When `strict` is `true`, the `ItemStack` is checked for an exact match. Otherwise, if only the components specified match, the ingredient will pass. This can be used by specifying the `type` as `neoforge:components`.
```

Comment:
```suggestion
`DataComponentIngredient`s compare the item, damage, and [data components][datacomponents] on an `ItemStack`. When `strict` is `true`, the `ItemStack` is checked for an exact match. Otherwise, if only the components specified match, the ingredient will pass. This can be used by specifying the `type` as `neoforge:components`.
```